doc/nrf/app_dev/device_guides/nrf70/fw_patches_ext_flash.rst

```diff
@@ -121,13 +104,16 @@ The external memory partition name must be defined in the devicetree or in the p
         device: MX25R64
         region: external_flash
 
+.. note::
+
+   This feature requires the partition manager to be enabled.
```

Comment:
I think we can mention the feature instead of "This feature"?

Reply:
Yes, but `note` highlights the importance.